### PR TITLE
Add a windows version of the launch-game script.

### PR DESCRIPTION
**Pull request by pchote**
The windows counterpart to #12992.

### CODE REVIEW

**Comment by reaperrr** [comment]
Is there a reason why this is placed in an `OpenRA` subfolder instead of the root folder like the other scripts?

**Comment by pchote** [comment]
There was indeed a reason, but it was not intended to make it though to the PR.  Fixed.

File: launch-game.cmd [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+OpenRA.Game.exe Game.Mod=%mod%
+if %errorlevel% neq 0 goto crashdialog
+exit
+:crashdialog
+echo ----------------------------------------
+echo OpenRA has encountered a fatal error.
+echo   * Log Files are available in Documents\OpenRA\Logs
+echo   * FAQ is available at https://github.com/OpenRA/OpenRA/wiki/FAQ
+echo ----------------------------------------
+pause
```
Comment (forcecore) [inline launch-game.cmd]: Nit: is it ok for .cmd not to have no EOL at end?

**Review by reaperrr — APPROVED** [review]

File: launch-game.cmd [hunk 2]
```diff
@@ -0,0 +1,24 @@
+@echo off
+title OpenRA
+:choosemod
+set /P mod=Select mod (ra, cnc, d2k, ts) or --exit:
```
Comment (Phrohdoh) [inline launch-game.cmd]: Why would someone ever run `launch-game.cmd --exit`?

Comment (obrakmann) [inline launch-game.cmd]: Same question as in #12992, really. How would this handle custom mods?

File: launch-game.cmd [hunk 3]
```diff
@@ -0,0 +1,24 @@
+@echo off
+title OpenRA
+:choosemod
+set /P mod=Select mod (ra, cnc, d2k, ts) or --exit: 
+if /I "%mod%" EQU "--exit" (exit)
+if /I "%mod%" EQU "ra" (goto launch)
+if /I "%mod%" EQU "cnc" (goto launch)
+if /I "%mod%" EQU "ts" (goto launch)
```
Comment (obrakmann) [inline launch-game.cmd]: And again as in #12992, do we want to add that already?

File: launch-game.cmd (hunk 2 above)
Comment (pchote) [inline launch-game.cmd]: Same answer as #12992 here and below.

**Review by obrakmann — APPROVED** [review]
👍

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/311725af00b0aa435d5318ce81b44259e09b7913)

**Pull request merged**